composer.json

```diff
@@ -9,9 +9,13 @@
     "require": {
         "phpoption/phpoption": "1.*"
     },
+    "require-dev": {
+        "phpunit/phpunit": "3.*"
```

Comment:
yeah, sorry,  habit, phpunit dropped too much functionality for us to use 4.x.  Will fix, should be fine here.